src/routes/mods.rs

```diff
@@ -15,6 +16,45 @@ pub async fn mod_search(
     Ok(HttpResponse::Ok().json(results))
 }
 
+#[derive(Serialize, Deserialize)]
+pub struct ModIds {
+    pub ids: String,
+}
+
+// TODO: Make this return the full mod struct
+#[get("mods")]
+pub async fn mods_get(
+    web::Query(ids): web::Query<ModIds>,
+    pool: web::Data<PgPool>,
+) -> Result<HttpResponse, ApiError> {
+    let mods_data =
+        database::models::Mod::get_many(serde_json::from_str::<Vec<i64>>(&*ids.ids)?, &**pool)
```

Comment:
This is incorrect;  The input for this should be a JSON array of mod ids represented as base62 strings.  (We could use integers, but we need to be consistent, and we use base62 everywhere else).

This also applies to the other "get multiple" routes; they should all take base62 ids.